packages/sdk-js/src/models/node/contractRead.ts

```diff
@@ -4,7 +4,7 @@ import * as avs_pb from "@/grpc_codegen/avs_pb";
 import { NodeType } from "@avaprotocol/types";
 
 // Required props for constructor: id, name, type and data: { contractAddress, callData, contractAbi, method }
-type ContractReadNodeData = avs_pb.ContractReadNode.AsObject;
+export type ContractReadNodeData = avs_pb.ContractReadNode.AsObject;
```

Comment:
The client side needs all these type exports to define objects such as `avs_pb.ContractReadNode.AsObject`